tests/Tools/generate-stub-map

```diff
@@ -205,7 +206,7 @@ use const PHP_EOL;
             continue;
         }
 
-        if (in_array($directoryInfo->getBasename(), ['tests', 'meta', 'vendor', 'couchbase_v2'], true)) {
+        if (in_array($directoryInfo->getBasename(), $excludedStubDirNames, true)) {
```

Comment:
Thanks to the move, tests/vendor will no longer conflict with the generator, so these can be removed from here.